src/elementary/ParticleTupleID/src/ctor.hpp

```diff
@@ -0,0 +1,24 @@
+/**
+ *  @brief Constructor
+ *
+ *  @param[in] id1           a first particle identifier
+ *  @param[in] id2           a second particle identifier
+ *  @param[in] identifiers   additional particle identifiers (optional)
+ */
+ template< typename... IDs,
+           typename = std::enable_if_t<
```

Comment:
I think we need to do more of this.

Reply:
SFINAE it!

For this identifier, I really wanted a constructor that takes any number of ParticleID for simplicity. Why create a vector of only 2 entries? Now, that doesn't mean that the vector based one is useless (a user could have scanned the output channels for every particle type).